ocs_ci/utility/powernodes.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.node import wait_for_nodes_status, get_worker_nodes, get_master_nodes
+from ocs_ci.ocs.ocp import wait_for_cluster_connectivity
+from ocs_ci.utility.utils import TimeoutSampler, exec_cmd
+from ocs_ci.utility.utils import get_ocp_version
+
+logger = logging.getLogger(__name__)
+
+
+class POWERNODES(object):
+    """
+    Wrapper for PowerNodes
+    """
+
+    isKVM = False
```

Comment:
Does this need to be defined here or can we stick to initializing in the `__init__`?

Reply:
Completed with the latest push. Removed global entry.